spec/classroom_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require_relative '../classroom'
+require_relative '../student'
+
+describe Classroom do
+  describe '#initialize' do
+    it 'sets the label attribute' do
+      classroom = Classroom.new('1A')
+      expect(classroom.label).to eq('1A')
+    end
+
+    it 'initializes the students array' do
+      classroom = Classroom.new('1A')
+      expect(classroom.students).to eq([])
+    end
+  end
```

Comment:
- [ ]  As required for the project, to add unit tests for all the methods in all the classes, kindly make sure you also implement tests for the `add_student` method.